### DISCUSSION TITLE
Configuring Command Line sensor using YAML has moved.

### DISCUSSION THREAD

**Discussion by aurimassiaulys** [custom-components/nordpool#330] [General]
you should fix new YAML  formating after HA update.

**Comment by Hellowlol** [comment]
I have no idea what you are talking about. Please send a pr to fix anything that is wrong in the documentation

**Comment by rdperkins** [comment]
I agree with Hellowlol. How do I know if I have the problem in my configuration.yaml? I have the following in that file:
sensor: !include sensor.yaml and binary_sensor.*_occupancy in the influxdb: section. Thanks for your help.

**Comment by Dstoofdijk** [comment]
I 'm facing same issue: Configuring Command Line sensor using YAML has moved.
Solution would be to change "YAML configuration to integration key"
Unclear how to act in this

**Comment by Chris123NT** [comment]
I have no idea how to fix this issue and the documentation offers zero guidance as well. Gotta love arbitrary changes with no instructions on how to fix it.

**Comment by Hellowlol** [comment]
For command line sensor you can check out this post https://community.home-assistant.io/t/configuring-command-line-sensor-using-yaml-has-moved-consult-the-documentation-to-move-your-yaml-configuration-to-integration-key-and-restart-home-assistant-to-fix-this-issue/579429/26
